src/Lean/Data/PersistentArray.lean

```diff
@@ -370,19 +370,15 @@ def mkPersistentArray {α : Type u} (n : Nat) (v : α) : PArray α :=
 @[inline] def mkPArray {α : Type u} (n : Nat) (v : α) : PArray α :=
   mkPersistentArray n v
 
-end Std
+end Lean
 
-open Std (PersistentArray PersistentArray.empty)
+open Lean (PersistentArray)
 
-def List.toPersistentArrayAux {α : Type u} : List α → PersistentArray α → PersistentArray α
+def List.toPArray' {α : Type u} (xs : List α) : PersistentArray α :=
```

Comment:
The use of `'` here is a bit awkward. Since we are adding things outside the Lean namespace here we have to be careful. We would like the name `List.toPArray` to stay open for Std, but we also don't want to lose dot notation for code inside the Lean package, so the compromise is to use `'` on these functions.